pallets/jobs/src/lib.rs

```diff
@@ -262,7 +267,12 @@ pub mod module {
 			ensure!(job.expiry > now, Error::<T>::JobAlreadyExpired);
 
 			// charge the user fee for job submission
-			let fee = T::JobToFee::job_to_fee(&job);
+			let processing_fee = T::JobToFee::job_to_fee(&job);
+			let ttl_u32: u32 =
```

Comment:
Make sure that ttl is not zero!

Reply:
Why should TTL not be able to be 0? Wouldn't that just mean on submission job gets removed?